internal/exec/util/blkid.go

```diff
@@ -69,18 +69,20 @@ type FilesystemInfo struct {
 	Label string
 }
 
-func GetFilesystemInfo(device string) (FilesystemInfo, error) {
+// If strict is true, fail if we find traces of more than one filesystem on
+// the device.
+func GetFilesystemInfo(device string, strict bool) (FilesystemInfo, error) {
```

Comment:
Bikeshed: maybe just flip this as `allow_ambivalent` since that's the term used throughout and in libblkid itself?